src/util/Platform.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Utils for Platform.ts.
+ *
+ * @license
+ * Copyright 2023 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { networkInterfaces } from 'os';
```

Comment:
This functionality is already in the existing Network class in an nodejs agnostic and enhanceable way

Reply:
perfect - I'll use that instead.